datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -3036,6 +3042,35 @@ private void configureDataProcessInstanceResolvers(final RuntimeWiring.Builder b
         "DataProcessInstance",
         typeWiring ->
             typeWiring
+                .dataFetcher(
```

Comment:
We can perhaps add an API smoke test for data process instance APIs so we can ensure that all of these fields are resolving correctly. Shirshanka may be able to help point you in the right direction but essentially this would live in smoke-tests/tests

Reply:
@RyanHolstien - ^